docs-kits/kits/Quality-Kit/Software Development View/page_software-development-view.md

```diff
@@ -11,7 +11,93 @@ sidebar_position: 4
 
 ## Quality Data Offers at EDC
 
-For the process described in the KIT Quality, data exchange between the participating partner companies is necessarily to be done for large vehicle and product populations. The data exchange therefore tshould be done as a file download via EDC according to the following specifications.
+When a Data Consumer calls the Catalog of a Data Provider, the Data Provider must signal in each Data Offer what exactly
+a Consumer could negotiate for. Data Offers in the Catalog are sorted by dcat:Datasets which are registered in the EDC-
+Management API as edc:Asset. Each Asset has private and public properties. The public properties are shown in the
+catalog and give hints to the Data Consumer what API and data they may expect. There are some properties that are
+mandatory for the entire Catena-X network and some that are mandatory only in specific Business Scenarios (like
+Quality).
+
+The dataAddress object's structure is determined by the dataplane implementation as it configures the details of the
+data transfer. They are not visible via the catalog.
+
+The following suggestion is a non-standardized draft how Assets (and thus by proxy, dcat:Datasets) should be registered
+in the Quality Use-Case.
+
+```json
+{
+  "@context": {
+    "cx-taxo": "https://w3id.org/catenax/taxonomy#",
+    "cx-common": "https://w3id.org/catenax/ontology/common#",
+    "dct": "https://purl.org/dc/terms/",
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@id": "someId",
+  "@type": "edc:Asset",
+  "edc:properties": {
+    "dct:type": {
+      "@id": "cx-taxo:ProductDescription"
+    },
+    "cx-common:version": "1.0",
+    "dct:language": {
+      "@id": "https://w3id.org/idsa/code/EN"
+    },
+    "dcat:qualifiedRelation": {
+      "dct:isPartOf": {
+        "@id": "http://my.quality/task"
+      }
+    },
+    "dct:conformsTo": {
+      "@id": "urn:samm:io.catenax.vehicle.product_description:3.0.0#ProductDescription"
+    },
+    "dct:description": "TBD",
```

Comment:
Description is not defined by Quality, right? We might remove it here if we have no good example.